.github/workflows/publish-chart.yaml

```diff
@@ -0,0 +1,34 @@
+name: Publish Chart
+
+on:
+  workflow_call:
+    inputs:
+      release_version:
+        description: "The release version to publish"
+        required: true
+        type: string
+      ref:
+        description: "The git ref to checkout"
+        required: true
+        type: string
+
+jobs:
+  publish-chart:
+    runs-on: ubuntu-22.04
+
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ inputs.ref }}
+      - uses: azure/setup-helm@v4
+        with:
+          version: "v3.0.2"
+      - run: |
+          helm plugin install https://github.com/chartmuseum/helm-push.git
```

Comment:
This will fail if runner already has it. It's not very likely that they will add it to the image, but it's a quick check.

```bash
➜ helm plugin install https://github.com/chartmuseum/helm-push.git
Error: plugin already exists
loft-enterprise [ main][$]
✗ echo $?
1
```